www/www-jekyll.yml

```diff
@@ -1,12 +1,29 @@
-# Install NGINX, and prepare the OME (UoD/SLS) prerequisites
+# Update the static website
 
 - hosts: www
-  environment:
-      PATH: /usr/local/bin:{{ ansible_env.PATH }}
+
+  # TODO: Remove these pre-tasks after the new archive is deployed to
+  # production for the first time
+  pre_tasks:
+
+  - name: Check for old deployment directory
+    stat:
+      path: "{{ deploy_archive_symlink }}"
+    register: _archive_symlink_st
+
+  - name: Rename old deployment directory if present
+    become: yes
+    command: mv "{{ deploy_archive_symlink }}" "{{ deploy_archive_symlink }}.old-jekyll-deploy"
+    when: _archive_symlink_st.stat.exists and _archive_symlink_st.stat.isdir
+
   roles:
-    - role: openmicroscopy.jekyll-build
-      tags: jekyll
-      jekyll_build_git_repo: "https://github.com/openmicroscopy/www.openmicroscopy.org"
-      jekyll_build_force_rebuild: False
-      jekyll_build_config: ['_config.yml', '_prod.yml']
-      jekyll_build_name: "{{ website_name }}"
+  - role: openmicroscopy.deploy-archive
+    become: yes
+    tags: jekyll
+
+  vars:
+    website_version: 20180711
+    deploy_archive_dest_dir: /var/www/{{ website_name }}/{{ website_version }}
```

Comment:
I've left the `website_name` variable unchanged (defined in mgmt-tools), can anyone remember the reason for using it instead of hardcoding the webiste directory `www.openmicroscopy.org`?